pkgs/tools/text/ispell/default.nix

```diff
@@ -5,7 +5,10 @@ stdenv.mkDerivation rec {
   version = "3.4.00";
 
   src = fetchurl {
-    url = "http://fmg-www.cs.ucla.edu/geoff/tars/${pname}-${version}.tar.gz";
+    urls = [
+      "https://www.cs.hmc.edu/~geoff/tars/${pname}-${version}.tar.gz"
+      "http://fmg-www.cs.ucla.edu/geoff/tars/${pname}-${version}.tar.gz"
+    ];
```

Comment:
We don't need a not working mirror. lets switch to https://www.cs.hmc.edu/~geoff/ispell.html fully.